### ISSUE TITLE
Next.js ReferenceError: window is not defined

### ISSUE THREAD

**Issue by trevor-trinh** [Simonwep/selection#127]
#### What is the problem?

This is my first github issue ever, so please let me know if you need more information. In a Next.js app, I wanted to use selection to select multiple items in a card. However, after installing viselect, I couldn't import SelectionArea from @viselect/react. 

#### What is the current behavior?

When I do 
`import SelectionArea, { SelectionEvent } from '@viselect/react';` 
Nextjs returns with:

>  "ReferenceError: window is not defined"

Screenshot of error: 
![image](https://user-images.githubusercontent.com/50549133/123716396-553c2680-d82f-11eb-936b-84da8f63bbce.png)

I tried to edit webpack as specified in another issue [here](https://github.com/JedWatson/react-select/issues/3802#issuecomment-589819173). I added a next.config.js with 
```
module.exports = {
  webpack: (config, { buildId, dev, isServer, defaultLoaders, webpack }) => {
    config.resolve.aliasFields = []
    return config;
  },
};
```
however the problem persisted. This is also in the CodeSandbox below.

#### Please provide the steps to reproduce and create a [CodeSandbox](https://codesandbox.io/).
Reproduce with a fresh install of Next.js using `npx create-next-app` and then `npm install @viselect/react`. 
[CodeSandbox](https://codesandbox.io/s/fervent-brook-t101o?file=/pages/index.js) with same exact behavior

#### What is the expected behavior?

I expect to not have the "window is not defined" error and be able to import use the component.

#### Your environment:
```
Toolset (e.g. webpack, vite, vue-cli...): next.js
Version (@viselect/<insert package>): @viselect/react
Browser:  Chrome
OS:  Windows 10
```

**Comment by Simonwep** [comment]
Seems to be related to server-side rendering... Solution seems to be [this SO thread](https://stackoverflow.com/questions/55151041/window-is-not-defined-in-next-js-react-app). I'll take a look into it later, thank your for trying the beta version! :)

**Comment by Simonwep** [comment]
Fixed in [v3.0.0-beta.3](https://github.com/Simonwep/selection/releases/tag/v3.0.0-beta.3) :)
